### DISCUSSION TITLE
Welcome to graphQL Discussions!

### DISCUSSION THREAD

**Discussion by xieguigang** [xieguigang/graphQL#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SweetPezzy** [comment]
I just got started with this stuff

**Comment by SweetPezzy** [comment]
How do I know if ya want to go further with this shit.
